samples/bot-ai-meeting-helper/.localConfigs

```diff
@@ -0,0 +1,22 @@
+BOT_ID=
+BOT_PASSWORD=
+AZURE_OPENAI_API_KEY=
+AZURE_OPENAI_ENDPOINT=
+AZURE_OPENAI_DEPLOYMENT_NAME=
+Token=
+BOT_ENDPOINT=
+Base64EncodedCertificate=
+EncryptionCertificateId=
+PRIVATE_KEY_PATH=
+OPENSSL_CONFIG=C:/Program Files/OpenSSL-Win64/bin/openssl.exe
```

Comment:
This looks hardcoded. Please check if you have already mentioned in readme for this.
How to get this exe?
Any link is added in readme ?

Reply:
removed